app/controllers/api/functions.php

```diff
@@ -1063,6 +1069,7 @@
     ->desc('Retry Build')
     ->label('scope', 'functions.write')
     ->label('event', 'functions.[functionId].deployments.[deploymentId].update')
+    ->label('audits.resource', 'function/{request.functionId}/deployment/{request.deploymentId}/build/{request.buildId}')
```

Comment:
Again, all logs can be saved on the functions scope.
```suggestion
    ->label('audits.resource', 'function/{request.functionId}')
```